recurly/__init__.py

```diff
@@ -1504,6 +1504,39 @@ class CreditPayment(Resource):
         'voided_at',
     )
 
+
+class ExportDate(Resource):
+    nodename = 'export_date'
+    collection_path = 'export_dates'
+
+    attributes = ('date', 'export_files')
+
+    def files(self, date):
+        """
+        Fetch files for a given date.
+        :param date: The date to fetch the export files for
+        :return: A list of exported files for that given date or an empty list if not file exists for that date
+        """
+        url = urljoin(recurly.base_uri() + self.collection_path, '/%s/export_files' % date)
+        return ExportDateFile.paginated(url)
+
+
+class ExportDateFile(Resource):
+    nodename = 'export_file'
+    collection_path = 'export_files'
+
+    attributes = ('name', 'md5sum', 'expires_at', 'download_url')
```

Comment:
Same nit here

Reply:
Sure, changed in this commit https://github.com/recurly/recurly-client-python/pull/288/commits/1d05a25c3d3fca164886e9c26bd949ce7ce5af44